### PR TITLE
Add option to query according to update title

### LINKED ISSUE #251: add option to query according to update title
If I know the update title, there seems to be no way to retrieve the corresponding update object. There is no `query(title=)` or similar. Can you please implement this?

From IRC:

```
<threebean> kparal: can you file an RFE to add a 'title' to the ListUpdatesSchema ?
```

And a follow-up:

```
<lmacken> kparal: like=title should work?
<lmacken> but we should have a proper title field
<kparal> lmacken: that works!
```

python-fedora 0.5.5

### PR DESCRIPTION
**Pull request by sebwoj**
Fix #251

### CODE REVIEW

File: bodhi/tests/client/test_bindings.py [hunk 1]
```diff
@@ -817,6 +817,19 @@ def test_with_page(self):
         client.send_request.assert_called_once_with(
             'updates/', verb='GET', params={'packages': 'bodhi', 'page': 5})
 
+    def test_with_titile(self):
```
Comment (Zlopez) [inline bodhi/tests/client/test_bindings.py]: typo titile -> title

**Review by Zlopez — COMMENTED** [review]
Only found a small typo

**Comment by bowlofeggs** [comment]
Two notes:

0. This actually does not fix #251 because that issue is about adding a title search to the REST API, not just to the Python bindings.
1. We are planning to drop the title field anyway in order to fix #1542, so we don't really need to fix #251.

We will likely delete the title field soon (in the next few months) so this patch won't be in Bodhi for very long, however, it does work so I see no reason not to accept it. I am going to rebase,d squash your commits, and then merge. Thanks for the patch!

**Review by bowlofeggs — APPROVED** [review]

**Comment by bowlofeggs** [comment]
Mergify seems stuck again, so I will manually merge this.

**Comment by bowlofeggs** [comment]
This patch is planned for inclusion in the upcoming 4.0.0 release: #3221

**Comment by bowlofeggs** [comment]
Bodhi 4.0.0 beta is built and deployed to staging:

https://copr.fedorainfracloud.org/coprs/bowlofeggs/bodhi-pre-release
https://bodhi.stg.fedoraproject.org/

**Pull request merged**